ddtrace/contrib/pylons/middleware.py

```diff
@@ -41,7 +41,11 @@ def _start_response(status, *args, **kwargs):
             except Exception as e:
                 # "unexpected errors"
                 # exc_info set by __exit__ on current tracer
-                span.set_tag(http.STATUS_CODE, getattr(e, 'code', 500))
+                code = getattr(e, 'code', 500)
+                if isinstance(code, int) and code >= 100 and code < 600:
```

Comment:
oh but it is not, because it can be changed by developers (that's the reason of this PR right?). Indeed if you:
```python
    def start_response_exception(self, status, headers):
        e = Exception("Some exception")
        e.code = '512'
        raise e

# and in the test
sc = int(s.get_tag("http.status_code"))
eq_(sc, 512)
```

it will fail. Here you can easily switch back to a `500` if a non digit string is here, otherwise you can cast and check the range. Right?